kwanj-k/storemanager-v2
# #161437270 Add tests and folder structure

### What does this PR do?
Adds tests for the version two of the API 

#### Description of Task to be Completed ?
* Create the project folder structure
* Write tests for: 
-- Add signup endpoint and login
-- Add/get and delete product endpoint
-- Add admin and attendant endpoint

### How should this be manually tested?
* After cloning the repo,git pull the branch ch-tests- #161437270
* Install a virtual enviroment and activate it [Refer here](https://docs.python-guide.org/dev/virtualenvs/)
* Pip install the the requirements.txt
* Run the following:

  ``` export FLASK_APP='run.py' ```
  ``` export APP_SETTINGS='development' ```
   ``` pytest --cov-report term-missing --cov=app ```

The new 14 tests should fail as the functionality is not there

![screenshot from 2018-10-24 05-41-40](https://user-images.githubusercontent.com/33450849/47402919-3452d780-d750-11e8-8908-a86e78ae9e97.png)


### What are the relevant Pivotal Tracker stories?
  #161437270
